searx/settings.yml

```
@@ -1563,6 +1563,19 @@ engines:
   #    engine: scanr_structures
   #    disabled: true
 
+  - name: searchmysite
+    engine: xpath
+    shortcut: sms
+    categories: general
+    paging: true
+    search_url: https://searchmysite.net/search/?q={query}&page={pageno}
+    url_xpath: //p[@class='sms-p3']/a[@class='result-link']/text()
+    title_xpath: //span[@class='result-title-txt']/text()
```

```suggestion
    title_xpath: //span[contains(@class,'result-title-txt'])/text()
```
makes it more robust against changes at their side